### ISSUE TITLE
Re-classification of wild barley

### ISSUE THREAD

**Issue by MikePapaWhisky** [vbonhomme/GrMoEx#65]
I'm not sure how this will be presented in GrMoEx, but we'd like an option to re-classify the ````taxa```` factor for ````Hspo```` grains. The re-classifications will be based on the traditional dimensions breadth (````width```` in the ```Traditional```` tab) and thickness (````height````). The Hspo group will be fit into four categories:

+ Width =< 2.25mm AND Thickness =< 1.5mm
+ Width =< 2.25mm AND Thickness > 1.5mm
+ Width > 2.25mm AND Thickness =< 1.5mm
+ Width > 2.25mm AND Thickness > 1.5mm

**Comment by vbonhomme** [comment]
I will do it manually, otherwise will be a big mess.

2016-01-14 12:44 GMT+00:00 Michael Paul Wallace <notifications@github.com>:

> I'm not sure how this will be presented in GrMoEx, but we'd like an option
> to re-classify the taxa factor for Hspo grains. The re-classifications
> will be based on the traditional dimensions breadth (width in the
> Traditionaltab) and thickness (height`). The Hspo group will be fit into
> four categories:
>
>    - Width =< 2.25mm AND Thickness =< 1.5mm
>    - Width =< 2.25mm AND Thickness > 1.5mm
>    - Width > 2.25mm AND Thickness =< 1.5mm
>    - Width > 2.25mm AND Thickness > 1.5mm
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vbonhomme/GrMoEx/issues/65>.
>



-- 
vincentbonhomme.fr <http://www.vincentbonhomme.fr>

**Comment by MikePapaWhisky** [comment]
Thanks. Let me know when you have something. There is a good reason for it,
I promise!

**Comment by vbonhomme** [comment]
yeah, degrading a continuous variable into classes, that's always the
promise of something amazing

2016-01-17 12:57 GMT+01:00 Michael Paul Wallace <notifications@github.com>:

> Thanks. Let me know when you have something. There is a good reason for it,
> I promise!
> On 14 Jan 2016 18:23, "Vincent Bonhomme" <notifications@github.com> wrote:
>
> > I will do it manually, otherwise will be a big mess.
> >
> > 2016-01-14 12:44 GMT+00:00 Michael Paul Wallace <
> notifications@github.com
> > >:
> >
> > > I'm not sure how this will be presented in GrMoEx, but we'd like an
> > option
> > > to re-classify the taxa factor for Hspo grains. The re-classifications
> > > will be based on the traditional dimensions breadth (width in the
> > > Traditionaltab) and thickness (height`). The Hspo group will be fit
> into
> > > four categories:
> > >
> > > - Width =< 2.25mm AND Thickness =< 1.5mm
> > > - Width =< 2.25mm AND Thickness > 1.5mm
> > > - Width > 2.25mm AND Thickness =< 1.5mm
> > > - Width > 2.25mm AND Thickness > 1.5mm
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/vbonhomme/GrMoEx/issues/65>.
> > >
> >
> >
> >
> > --
> > vincentbonhomme.fr <http://www.vincentbonhomme.fr>
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/vbonhomme/GrMoEx/issues/65#issuecomment-171729619>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vbonhomme/GrMoEx/issues/65#issuecomment-172317651>.
>



-- 
vincentbonhomme.fr <http://www.vincentbonhomme.fr>

**Comment by MikePapaWhisky** [comment]
We're trying to ascertain whether people are misidentifying barley grains.
What would be interesting is that if the large (typical of domestics)
grains identified as wild had the same shape as the other domestics. If
archaeology is detective work, we are the police oversight commission.

**Comment by vbonhomme** [comment]
* :+1: for the joke but tired of all your :hankey: 
* if you're not :smile: call the :cop:
* gonna :metal: it for :candy: 
* from :fr: with :heart:
